docs/essentials/collections/threadsafe/blockingCollectionOverview.md

```diff
@@ -1 +1,115 @@
+# BlockingCollection Overview
+
+[BlockingCollection&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Concurrent.BlockingCollection%601.html) is a thread-safe collection class that provides the following features:
+
+*   An implementation of the Producer-Consumer pattern.
+
+*   Concurrent adding and taking of items from multiple threads.
```

Comment:
@ cartermp You're right - the meaning of Take() is different than in LINQ, which indeed makes this more confusing. I'll change it to adding and removing.